statsmodels/genmod/families/family.py

```diff
@@ -1432,3 +1432,193 @@ def resid_anscombe(self, endog, mu):
         return ((hyp2f1(-self.alpha * endog) - hyp2f1(-self.alpha * mu) +
                  1.5 * ( endog**(2 / 3.) - mu**(2 / 3.))) /
                 (mu + self.alpha * mu**2)**(1 / 6.))
+
+
+class Tweedie(Family):
+    """
+    Tweedie family.
+
+    Parameters
+    ----------
+    link : a link instance, optional
+        The default link for the Tweedie family is the log link when the
+        link_power is 0. Otherwise, the power link is default.
+        Available links are log, cloglog, identity, nbinom and power.
+        See statsmodels.family.links for more information.
+    var_power : float, optional
+        The variance power.
+    link_power : float, optional
```

Comment:
Ohh... you stumbled across something I noticed a while back, but never bothered to ask. I was copying from the other classes. I think it should really say `link : a link class, optional`. `link` is actually instantiated in the `__init__` function. 

Reading through statsmodels docs and examples we've always told people to say something like `sm.GLM(endog=y, exog=x, family=sm.families.Gamma(link=sm.families.links.Log))`

Is that the best approach? Personally, I agree with you it seems more natural to pass an instance of a class, not the class itself.